### DISCUSSION TITLE
DateRange Picker [PRO] - Access Events when dates are cleared either by the cleaner or manual removal.

### DISCUSSION THREAD

**Discussion by ITSM-JonP** [coreui/coreui#780] [Q&A]
I'm trying to integrate the DateRange  picker (https://coreui.io/bootstrap/docs/forms/date-range-picker/) into a project. For each picker I have a pair  of hidden input fields that are populated via the _endDateChange.coreui.date-range-picker_ and _startDateChange.coreui.date-range-picker_ event. These fields have on change event listeners attached which then calls an AJAX api with the values. That part work fine.

However, when clearing the dates the paired hidden fields are not updated. This is the same regardless if the date text is removed from the rendered text field or via the "cleaner" button. 

What is the best way to update dependent fields when dates are cleared?

**Comment by mrholek** [comment]
Please upgrade @coreui/core-pro to v5.2.1; now `endDateChange` and `startDateChange` events emit `null` when you reset the component.
